### DISCUSSION TITLE
Cant add field to Taxonomy in graphql

### DISCUSSION THREAD

**Discussion by lotarbo** [statamic/cms#9021] [Troubleshooting]
Hello, i try to add field to Taxonomy type like in example https://statamic.dev/graphql#custom-fields
```
GraphQL::addField(TaxonomyType::NAME, 'terms', function () {
            return [
                'type' => GraphQL::string(),
                'resolve' => function () {
                    return 'test';
                }
            ];
        });
```

but it doesn't work.
Is there any reason why i can't add a field for the TaxonomyType ?

**Comment by duncanmcclean** [comment]
The `TaxonomyType` class isn't an interface, like entries, terms, assets, etc. You can only add fields to these interfaces:

https://github.com/statamic/cms/blob/45590cef0d70f80150db5f3789dd6e25cb1d95eb/src/GraphQL/TypeRegistrar.php#L66-L71 

If this is something you'd like to see added, please open a [feature request](https://github.com/statamic/ideas/issues/new).
